db/migrate/20170323164658_add_has_many_belongs_to_relationship_between_bet_and_debt.rb

```diff
@@ -0,0 +1,8 @@
+class AddHasManyBelongsToRelationshipBetweenBetAndDebt <
+  ActiveRecord::Migration[5.0]
+  def change
```

Comment:
You should be able to do this with one line:
`add_reference :debts, :bet, foreign_key: true`

This creates the column, foreign key constraint, and by default adds an index as well:
http://edgeapi.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_reference